src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -86,9 +73,11 @@ form {
 
 .body {
   margin: 0 32px;
-  padding-right: 24px;
+  padding: 32px;
   flex-grow: 1;
   overflow-y: auto;
+  color: var(--kd-color-text-level-secondary);
+  background-color: var(--kd-color-background-accent-subtle);
```

Comment:
I don't think we actually want a background color on the body here. I think it's there in Figma just to indicate it's a slot. Can double-check with Chadan, but it seems off to me.